common/men_at_arms_types/temp_draft_doc.txt

```diff
@@ -0,0 +1,587 @@
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = drakonid
+}
+
+##Nerubian
+arachnathid = { 
+	type = skirmishers
+	
+	damage = 14
+	toughness = 15
+	pursuit = 12
+	screen = 10
+	
+	terrain_bonus = {
+		taiga = { damage = 5 toughness = 3 pursuit = 5 screen = 5 }
+	}
+	
+	winter_bonus = {
+		mild_winter = { damage = 2 toughness = 2 pursuit = 2 screen = 2 }
+		normal_winter = { damage = 3 toughness = 3 pursuit = 3 screen = 3 }
+		harsh_winter = { damage = 4 toughness = 4 pursuit = 4 screen = 4 }
+	}
+
+	counters = {
+		heavy_infantry = 1
+	}
+
+	buy_cost = { gold = cost_55_recruitment_cost }
+	low_maintenance_cost = { gold = cost_55_low_maint_cost }
+	high_maintenance_cost = { gold = cost_55_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = arachnathid
+}
+jormungar_tunneler = {
+	type = siege_weapon
+	fights_in_main_phase = no
+	
+	damage = 0
+	toughness = 10
+	
+	siege_tier = 2
+	siege_value = 1.7
+	
+	buy_cost = { gold = cost_115_recruitment_cost }
+	low_maintenance_cost = { gold = cost_115_low_maint_cost }
+	high_maintenance_cost = { gold = cost_115_high_maint_cost }
+	
+	stack = 10
+	icon = jormungar_tunneler
+}
+
+jormungar_wyrm = { 
+	type = massive_infantry
+	
+	damage = 125
+	toughness = 140
+	pursuit = 5
+	screen = 35
+	
+	terrain_bonus = {
+		taiga = { damage = 20 toughness = 20 pursuit = 25 screen = 25 }
+	}
+	
+	winter_bonus = {
+		mild_winter = { damage = 5 toughness = 5 pursuit = 5 screen = 5 }
+		normal_winter = { damage = 5 toughness = 5 pursuit = 5 screen = 5 }
+		harsh_winter = { damage = 5 toughness = 5 pursuit = 5 screen = 5 }
+	}
+
+	counters = {
+		skirmishers = 1
+	}
+
+	buy_cost = { gold = cost_180_recruitment_cost }
+	low_maintenance_cost = { gold = cost_180_low_maint_cost }
+	high_maintenance_cost = { gold = cost_180_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = jormungar_wyrm
+}
+
+venomancer = {
+	type = caster
+	
+	damage = 55
+	toughness = 25
+	pursuit = 30
+	screen = 30
+	
+	terrain_bonus = {
+		taiga = { damage = 20 toughness = 20 pursuit = 25 screen = 25 }
+	}
+
+	winter_bonus = {
+		mild_winter = { damage = 5 toughness = 5 }
+		normal_winter = { damage = 5 toughness = 5 }
+		harsh_winter = { damage = 5 toughness = 5 }
+	}
+
+	counters = {
+		pikemen = 1
+		massive_infantry = 2
+		heavy_infantry = 2
+	}
+
+	buy_cost = { gold = cost_120_recruitment_cost }
+	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = venomancer
+}
+
+webspinner = {
+	type = archers
+	
+	damage = 35
+	toughness = 20
+	pursuit = 35
+	screen = 20
+	
+	terrain_bonus = {
+		mountains = { damage = 12 toughness = 10 }
+		taiga = { damage = 12 toughness = 10 }
+	}
+
+	counters = {
+		skirmishers = 1
+		light_aerial = 1
```

Comment:
I'd give them bonus against cavalry as well, because flying sticky nets can bring a charge to a hold pretty easy.